exp/tests/test_response_views.py

```diff
@@ -288,7 +288,6 @@ def test_delete_preview_data(self):
         self.assertEqual(
             self.study.responses.filter(is_preview=True).count(), self.n_previews
         )
-        response = self.client.post(url, {})
```

Comment:
this shouldn't be removed - the assigned output isn't used, but making the post request is important

Reply:
Agreed.  That shouldn't have been removed.  It was removed manually by me, My apologies. It should be back in the code base now.